src/Raven.Server/Documents/PeriodicBackup/GoogleCloud/RavenGoogleCloudClient.cs

```diff
@@ -149,6 +149,15 @@ public Task<Object> GetObjectAsync(string fileName)
             );
         }
 
+        public async Task<Size> GetObjectSizeAsync(string fileName)
+        {
+            var obj = await GetObjectAsync(fileName);
```

Comment:
you cannot get just the size? This looks like it might be downloading the object, just to get the size. You probably can get just metadata?